lib/HTFeed/Stage/ImageRemediate.pm

```diff
@@ -186,86 +203,82 @@ sub stage_info {
     return { success_state => 'images_remediated', failure_state => '' };
 }
 
+
 sub _remediate_tiff {
     my $self                     = shift;
     my $infile                   = shift;
     my $outfile                  = shift;
     my $force_headers            = ( shift or {} );
     my $set_if_undefined_headers = shift;
+    my $check_jhove_status       = shift;
```

Comment:
I think this might have been a direction that we didn't end up using? Not 100% sure